build.gradle

```diff
@@ -248,9 +247,9 @@ task pythonFormatterPreCommit() {
 
 task python2PostCommit() {
   dependsOn ":sdks:python:test-suites:portable:py2:crossLanguagePythonJavaKafkaIOFlink"
-  dependsOn ":sdks:python:test-suites:portable:py2:crossLanguageTests"
+  dependsOn ":sdks:python:test-suites:portable:crossLanguageTests"
```

Comment:
I'm not sure I follow this change - this is currently still a python2 PostCommit suite, so it should include py2 only tests. 

However you could convert the targets that are not version-specific, such as
```
task portablePythonPreCommit() 
task pythonSparkPostCommit() 
```
to point to portable/build.gradle

Reply:
restored crossLanguageTests and directRunnerIT to py2 only.